services/vault-watcher/clarifai_vault_watcher/main.py

```diff
@@ -0,0 +1,50 @@
+"""
+ClarifAI Vault Watcher Service
+
+This service monitors the vault directory for changes in Markdown files
+and emits dirty block IDs for processing.
+"""
+
+import os
+import time
+import logging
+
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+)
+logger = logging.getLogger(__name__)
+
+
+def main():
+    """Main entry point for the Vault Watcher service."""
+    logger.info("Starting ClarifAI Vault Watcher service...")
+    
+    # Check environment variables
+    vault_path = os.getenv('VAULT_PATH', '/vault')
+    rabbitmq_host = os.getenv('RABBITMQ_HOST', 'rabbitmq')
+    
+    logger.info(f"Watching vault path: {vault_path}")
+    logger.info(f"RabbitMQ host: {rabbitmq_host}")
+    
+    # Ensure vault directory exists
+    if not os.path.exists(vault_path):
+        logger.warning(f"Vault path does not exist: {vault_path}")
+        os.makedirs(vault_path, exist_ok=True)
+        logger.info(f"Created vault directory: {vault_path}")
+    
+    # Main service loop (placeholder)
+    try:
+        while True:
```

Comment:
again, we need to indicate change in service state more than each iteration of the loop (the log is happening every time no matter what)